src/website/app/demos/Card/examples/card/clickable.js

```diff
@@ -24,17 +24,19 @@ export default {
   title: 'Clickable',
   // $FlowFixMe
   backgroundColor: mineralTheme.color_gray_10,
-  description:
-    'If an onClick function is provided, the entire card becomes clickable and keyboard actionable.',
+  description: `If an \`onClick\` callback is provided, the entire card becomes clickable and keyboard actionable.
+
+Use this feature when the Card represents a [media object](http://www.stubbornella.org/content/2010/06/25/the-media-object-saves-hundreds-of-lines-of-code/) or if only one action can be taken per Card.
```

Comment:
I don't understand this guidance. How does a CSS media object relate to a Card?

Reply:
yeah, the media object reference is probably a bit too specific. I was thinking of a Card that defines one or more actions vs a Card that represents a row in a db. I'll just take this out.